examples/text_classification/model.py

```diff
@@ -1,5 +1,15 @@
 import torch.nn as nn
 
+r'''
+The model is composed of the embeddingbag layer and the linear layer.
+nn.EmbeddingBag computes the mean of 'bags' of embeddings. Since it
+doesn't instantiate the intermediate embeddings, nn.EmbeddingBag can
```

Comment:
I think you mean "since it accumulate the average across the embeddings on the fly"